### DISCUSSION TITLE
Can't use OpenAI API tries to use Ollama instead

### DISCUSSION THREAD

**Discussion by byjlw** [open-webui/open-webui#4445] [General]
Ran this command but with a real key as explained in the readme
`docker run -d -p 3000:8080 -e OPENAI_API_KEY=your_secret_key -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main`

What i expected was a list of openAI API models

What i got was the list of models from my ollama instance.

**Comment by justinh-rahb** [comment]
Check what's in your OpenAI Connection settings:

<img width="767" alt="Screenshot 2024-08-07 at 7 13 03 PM" src="https://github.com/user-attachments/assets/4ae4ccb9-0fca-45c7-a6a2-4f9444be86fc">
